### PR TITLE
chore(fuselage): `Button` size

### PR DESCRIPTION
**Pull request by juliajforesti**
## Proposed changes (including videos or screenshots)

- change `Button` default size to 40px

### CODE REVIEW

File: packages/fuselage/src/components/ButtonGroup/ButtonGroup.spec.tsx [hunk 1]
```diff
@@ -1,13 +1,22 @@
 import { composeStories } from '@storybook/testing-react';
 import { render } from '@testing-library/react';
+import { axe, toHaveNoViolations } from 'jest-axe';
 import React from 'react';
 
 import * as stories from './ButtonGroup.stories';
 
 const { Default } = composeStories(stories);
```
Comment (tassoevan) [inline packages/fuselage/src/components/ButtonGroup/ButtonGroup.spec.tsx:8]: Jest extensions should be in a setup file, no?

File: packages/fuselage/src/components/Button/Button.tsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ export const Button = forwardRef(function Button(
     children,
     ...props
   }: ButtonProps,
-  ref: Ref<HTMLElement>
+  ref: Ref<HTMLButtonElement>
```
Comment (tassoevan) [inline packages/fuselage/src/components/Button/Button.tsx:44]: What if `<Box is='a' ref={ref} />`?

File: packages/fuselage/src/components/Button/Button.spec.tsx [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
+import { axe, toHaveNoViolations } from 'jest-axe';
 import React from 'react';
 
 import * as stories from './Button.stories';
+import * as iconButtonStories from './IconButton.stories';
 
 const { Default, AsIconButton } = composeStories(stories);
+const { _IconButton, _IconButtonInfo, _IconButtonSuccess } =
+  composeStories(iconButtonStories);
+expect.extend(toHaveNoViolations);
```
Comment (tassoevan) [inline packages/fuselage/src/components/Button/Button.spec.tsx:12]: Jest extensions should be in a setup file, no?

**Review by tassoevan — CHANGES_REQUESTED** [review]

File: packages/fuselage/src/components/Button/Button.tsx (hunk 2 above)
Comment (juliajforesti) [reply packages/fuselage/src/components/Button/Button.tsx]: you're right ✅ 
added ` | HTMLAnchorElement` type to ref and `AllHTMLAttributes<HTMLButtonElement | HTMLAnchorElement>`

**Pull request merged**